src/main/java/ast/psi/NodeFactory.java

```diff
@@ -0,0 +1,41 @@
+package ast.psi;
```

Comment:
Now I've create a new `NodeFactory` that just creates a NodeConfig and returns a new instance of a Node class. The builder class will use this to create the new instances